build.gradle

```diff
@@ -185,6 +185,7 @@ def startServerInBackground = tasks.register('startServerInBackground', com.gith
     args = ['--config', './frontend/cypress/config', '--since', 'd1', '--view']
     String versionJvmArgs = '-Dversion=' + getRepoSenseVersion()
     jvmArgs = [ versionJvmArgs ]
+    killDescendants = false // Needed because Java 8 does not support killDescendants
```

Comment:
Is this change related to the `com.github.psxpaul.execfork` upgrade?

Reply:
This flag had been implemented in version `0.1.13`, but it won't work with Java 8. Considering that `0.1.13` was not compatible with Gradle 7 anyway, I decided to use `0.1.12` before `0.2.0` came out.

As a side note, `0.1.14` was the first version to have Gradle 7 compatibility but it still could not work with Java 8. `0.2.0` was the first version to re-add Java 8 compatibility.

Reply:
Thanks for the clarification. This looks super confusing, nice work keeping track of everything! :sweat_smile: